### PR TITLE
Fix sits_coverage function

### LINKED ISSUE #119: Bug in sits_coverage function
`sits_coverage()` function is not working properly with WTSS coverages other than "MOD13Q1". Example:
```
sits_services()
# shows 'MOD13Q1' and 'MOD13Q1_M' (a new coverage for MOD13Q1 quality bands)
sits_coverage("WTSS-INPE", "MOD13Q1")
# it's working!
sits_coverage("WTSS-INPE", "MOD13Q1_M")
# fail!
```
Some possible related issues: #109

### LINKED ISSUE #121: sits_services() does not query WTSS service to show coverages
sits_services function does not query WTSS service to show coverage details.
It reads sits config to get bands and coverages.

### PR DESCRIPTION
**Pull request by rolfsimoes**
Resolve #119. Fix #121.

### CODE REVIEW

**Comment by rolfsimoes** [comment]
Fix issue #121

File: R/sits_config.R [hunk 1]
```diff
@@ -212,6 +212,29 @@ sits_show_config <- function() {
     names(minimum_values) <- bands
     return(minimum_values)
 }
+#' @title Retrieve the maximum values for a given band
+#' @name .sits_get_maximum_values
+#' @author Gilberto Camara, \email{gilberto.camara@@inpe.br}
+#'
+#' @param service          Name of the product.
+#' @param bands            Vector of bands.
+#' @return The minimum values.
```
Comment (pedro-andrade-inpe) [inline R/sits_config.R]:
```suggestion
#' @return The maximum values.
```

File: R/sits_config.R [hunk 2]
```diff
@@ -212,6 +212,29 @@ sits_show_config <- function() {
...
+.sits_get_maximum_values <- function(service, bands) {
+    # create a string to query for the maximum values
+    maximum_values <- vector()
+    mv <- paste0(service,"_maximum_value")
```
Comment (pedro-andrade-inpe) [inline R/sits_config.R]:
```suggestion
    mv <- paste0(service, "_maximum_value")
```

**Comment by pedro-andrade-inpe** [comment]
@rolfsimoes, pay attention to the syntax to close issues (see above). It is always better to close the issue directly in a commit description when you close more than one issue in a PR.

**Comment by rolfsimoes** [comment]
I got it! :+1:

**Pull request merged**